Gruntfile.js

```diff
@@ -1,82 +1,82 @@
 module.exports = function(grunt) {
-    var istanbulJpm = require("istanbul-jpm");
+  var istanbulJpm = require("istanbul-jpm")
     // gross, put this in the process
-    process.env.coveragedir = require("os").tmpdir();
-    console.log(process.env.coveragedir);
-    grunt.initConfig({
-        eslint: {
-            files: '{lib,data,test}/**/*.js{,on}',
-            options: {
-                quiet: true
-            }
-        },
-        shell: {
-            addonLintTest: {
-                command: 'jpm xpi; addons-linter --output json --pretty *xpi | node scripts/addon-lint-consumer.js',
-            },
-            makeCoverageTest: {
-                command: "echo > test/z-ensure-coverage.js; git ls-tree -r HEAD --name-only lib | grep \"js$\" | xargs -I '{}' echo 'require(\"../{}\");' | egrep -v \"(jetpack|main.js)\" >> test/z-ensure-coverage.js",
-            },
-            makeTestEnv: {
-                command: 'rm -rf testing-env && mkdir testing-env && cd testing-env && cat ../.jpmignore ../.jpmignore-testing-env > .jpmignore && ln -s ../Gruntfile.js . && ln -s ../node_modules . && ln -s ../data . && ln -s ../coverage/instrument/lib . && ln -s ../package.json . && ln -s ../test .',
-            },
-            jpmTest: {
-                command: 'cd testing-env && jpm test',
-            },
-            jpmTestTravis: {
-                command: 'cd testing-env && jpm test -b /usr/local/bin/firefox',
-            }
-        },
-        instrument: {
-            files: 'lib/**/*.js',
-            options: {
-                lazy: false,
-                basePath: 'coverage/instrument',
-                instrumenter: istanbulJpm.Instrumenter
-            }
-        },
-        storeCoverage: {
-            options: {
-                dir: 'coverage/reports'
-            }
-        },
-        makeReport: {
-            src: 'coverage/reports/**/*.json',
-            options: {
-                type: 'lcov',
-                dir: 'coverage/reports',
-                print: 'detail'
-            }
-        }
-    });
+  process.env.coveragedir = require("os").tmpdir()
+  console.log(process.env.coveragedir)
 
-    grunt.loadNpmTasks('grunt-eslint');
-    grunt.loadNpmTasks('grunt-babel');
-    grunt.loadNpmTasks('grunt-shell');
-    grunt.loadNpmTasks('grunt-istanbul');
+  grunt.initConfig({
+    eslint: {
+      files: ".",
+      options: {
+        quiet: true
+      }
+    },
+    shell: {
+      addonLintTest: {
+        command: "jpm xpi; addons-linter --output json --pretty *xpi | node scripts/addon-lint-consumer.js",
```

Comment:
This was failing before because `addons-linter` wasn't installed globally on my machine, or in the package.json. So I added it to package.json as a devDep.